main/2021/CG-08-03.md

```diff
@@ -27,8 +27,16 @@ Installation is required, see the calendar invite.
     1. Review of action items from prior meeting.
     1. Update on [Branch Hinting](https://github.com/WebAssembly/branch-hinting) [20 min].
         1. Poll for phase 3
-    1. Summary of [interface-types/#135](https://github.com/WebAssembly/interface-types/issues/135) and discussion [30 min].
-        1. Poll for maintaining single list-of-USV `string` type
+    1. Summary of [interface-types/#135](https://github.com/WebAssembly/interface-types/issues/135) and discussion [40 min].
+        1. Poll for supporting [UTF-16](https://github.com/WebAssembly/interface-types/issues/136#issuecomment-861799460) lifting and lowering
+        1. Poll for supporting compact UTF-16 (Latin1) lifting and lowering
+        1. Potential formal objection: A "**single** list-of-USV `string` type" may conflict with our [high-level goals](https://webassembly.org/docs/high-level-goals/), with [what](https://webassembly.org/) we [communicate](https://www.w3.org/2020/03/webassembly-wg-charter), and [envisioned](https://www.youtube.com/watch?v=ZKydJPRosa0). In particular:
+           * "integrate well with the existing Web platform"
+           * "execute in the same semantic universe as JavaScript"
+           * "maintain the backwards-compatible nature of the Web"
+           * "promote other compilers and tools targeting WebAssembly"
+           * "WebAssembly **also** supports non-web embeddings" (but not primarily)
+        1. Poll to maintain a **single** list-of-USV `string` type - or otherwise - allow isolated surrogates to cross component boundaries (enables lossless transfer of `DOMString`)
```

Comment:
1. Could you add `(byte strings)` after `isolated surrogates`? I'm not sure if everybody is familiar with that terminology.
2. Could you add `at the cost of interoperability` after `DOMString`? It's a trade-off.

Reply:
1. In WebIDL there is the concept of `ByteString`, which due to its value range cannot contain isolated surrogates unless reinterpreted. The concept of surrogates, paired or isolated, is exclusive to 16-bit encodings.
2. For affected languages it enables interoperability without correctness cliffs. Similar trade-off, but different perspective. Ambiguous.